assets/main.scss

```diff
@@ -13,6 +13,12 @@ body {
 
 .center-text {
   text-align: center;
+  margin-bottom: 0;
+  font-size: 26px;
+  font-weight: bold;
+  text-decoration: none;
+  color: $primary-color;
+  padding: 8px 0;
```

Comment:
Perhaps we can rename this class name or create a separate class  to put these styles in. The class name is center text, yet a lot more is being done.